lib/enumerate_it/base.rb

```diff
@@ -116,7 +116,7 @@ def register_enumeration(values_hash)
       end
 
       def define_enumeration_constant(name, value)
-        const_set name.to_s.tr('-', '_').upcase, value
+        const_set name.to_s.tr('-', '_').upcase.delete(' '), value
```

Comment:
hey @Bialogs - how about replacing the space with a `_`?
I think it'd make more sense, wouldn't it?

Something like:
```ruby
...to_s.gsub(/[-\s]/, '_')...
```

Reply:
Yea that makes sense.

Reply:
@lucascaton 

While converting the space to `_` with match `/\s/` it also matches a newline. While newline is valid in a symbol, I'm not sure what your thoughts are on supporting this in the gem and wanted to let you know.

It may be out of scope for this pull request. I'm really only needing spaces in my current usage.

Therefore my recommendation is to use `/\p{blank}/` match instead of `/\s/` so that the same `NameError: wrong constant name ...` will appear which led me down this path.